en/config_mc/README.md

```diff
@@ -1,10 +1,141 @@
+
+Relevant topics are:
+
+- [Vehicle (Frame) Selection](../config/airframe.md) - Select vehicle type to match your frame.
+- [Actuator Configuration and Testing](../config/actuators.md) - Vehicle geometry, output mapping, motor configuration, testing.
+- [ESC Calibration](../advanced_config/esc_calibration.md) - Do between output mapping and motor configuration (topic above) for PWM and OneShot ESC.
+
+
+## Sensor Setup and Calibration
+
+PX4 most commonly relies on a magnetometer (compass) for direction information, a barometer for altitude, a gyroscope for attitude, an accelerometer for velocity calculations, and a GPS/GNSS for global position.
+Pixhawk flight controllers (and many others) have inbuilt magnetometer, accelerometer, gyroscope, and barometer.
+The inbuilt compass usually isn't particularly reliable, and it is common to also add an external compass (usually with a GPS for position).
+
+We first need to set the [Sensor Orientation](../config/flight_controller_orientation.md), informing PX4 how the autopilot (and its inbuilt sensors) and external compasses are oriented relative to the vehicle.
+Generally you'll orient towards the front of the vehicle and not have to set anything.
+Once that is done we need to calibrate the compass(es), gyroscope, and accelerometer.
+
+The core sensor setup is covered in these topics.
+
+- [Sensor Orientation](../config/flight_controller_orientation.md)
+- [Compass](../config/compass.md)
+- [Gyroscope](../config/gyroscope.md)
+- [Accelerometer](../config/accelerometer.md)
+
+
+PX4 can use other peripherals, such as distance sensors, optical flow sensors, traffic avoidance alarms, cameras, and so on. 
+
+- [Flight Controller Peripherals](../peripherals/README.md) - Setup specific sensors, optional sensors, actuators, and so on.
+
+:::note
+Barometers don't need calibration, and if you've mounted the flight controller level, there isn't much value in a [Level Horizon](../config/level_horizon_calibration.md) calibration either.
+
+Obviously you don't need to calibrate or configure anything that isn't present or used by PX4.
+For multicopter, that includes [airspeed sensors](../config/airspeed.md), which are not used for flight control.
+:::
+
+## GPS Setup
+
+<!-- TBD 
+
+Note that it is plugnplay except for RTK. Worth saying.
+
+-->
+
+## Manual Control Setup
+
+<!-- note that you will have one or the other -->
+
+Radio Control:
+
+- [Radio Controller (RC) Setup](../config/radio.md)
+- [Flight Mode Configuration](../config/flight_mode.md)
+  
+Joystick/GamePad:
+
+- [Joystick Setup](../config/joystick.md)
+
+## Safety Configuration
+
+- [Battery/Power Module Setup](../config/battery.md)
+- [Safety Configuration (Failsafes)](../config/safety.md)
+
+## Tuning
+
+<!-- 
+- Explain what you have to tune on PX4, what you can tune, and what each topic covers
+- I expect we should start with an exhaustive list of the tuning you could want to do - such as position tuning, etc. Do we have one?
```

Comment:
Note, so here we have rate tuning (autotune), rate/position etc in the basic manual, +airmode tuning in the advanced, + setpoint tuning. Also latency/filters. Is there a complete list of the tuning that someone might want to do "of this type"?

Reply:
I think we need to organize the tuning pages a bit more, but for now I find it okay to just link them all here.

Reply:
Yes. For me the most important things right now are to:
- Clarify when Autotune can be used (and what to do if not)
- Have a list of all the tuning we might reasonable want to do, so we can design our tuning approach.